paimon-core/src/main/java/org/apache/paimon/append/AppendOnlyWriter.java

```diff
@@ -210,6 +211,14 @@ public void close() throws Exception {
         sinkWriter.close();
     }
 
+    public void toBufferedWriter() throws Exception {
+        flush(false, false);
```

Comment:
if it is a buffered sink write, add the spillable is true.

We don't need to new writer.